src/api/v1/debug.ts

```
@@ -0,0 +1,19 @@
+import { Elysia } from "elysia";
+import { authMiddleware } from "../middleware/auth";
+
+export const debug = new Elysia({ prefix: "/debug" })
+  .use(authMiddleware)
+  .get("/auth", ({ headers, ...ctx }: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)